README.md

```diff
@@ -96,22 +96,22 @@ FileInfo programFile = FileLocator.GetFileStartingAtSolutionDirectory("SourcePro
 Methods to compress and decompress data using gzip:
 ```cs
 // Will return the compressed byte array
-var myCompressedByteArray = Gzip.Compress(myByteArray);
+byte[] myCompressedByteArray = Gzip.Compress(myByteArray);
 
 // Will return the compressed string as byte array, using ASCII for encoding
-var myCompressedStringASCII = Gzip.Compress(myString, Encoding.ASCII);
+byte[] myCompressedStringASCII = Gzip.Compress(myString, Encoding.ASCII);
 
 // Will return the compressed string as byte array, using UTF8 for encoding
-var myCompressedStringUTF8 = Gzip.Compress(myString);
+byte[] myCompressedStringUTF8 = Gzip.Compress(myString);
 
 // Will return the decompressed byte array
-var myDecompressedByteArray = Gzip.Decompress(myCompressedByteArray);
+byte[] myDecompressedByteArray = Gzip.Decompress(myCompressedByteArray);
```

Comment:
Wait are you decompressing a byte array into a byte array..?

Reply:
Not exactly sure what you mean, to get the decompressed data. Am I missing something?

Reply:
No you're right - I'm dumb.